pynuodb/encodedsession.py

```diff
@@ -101,10 +161,24 @@ def open_database(self, db_name, parameters, cp):
         self.putNull().putString(cp.genClientKey())
 
         self._exchangeMessages()
-        version = self.getInt()
+        protocolVersion = self.getInt()
         serverKey = self.getString()
         salt = self.getString()
-        return version, serverKey, salt
+        self.__connectionDatabaseUUID = self.getUUID()
+        
```

Comment:
I think you should update protocol.py to have the same defines as we have for JDBC and C++. Then you can do

if protocolVersion >= GET_CONNID_FROM_SERVER

etc.